src/File Formats/CASC.cpp

```diff
@@ -40,8 +40,8 @@ namespace casc {
 		close();
 	}
 
-	void CASC::open(const fs::path& path) {
-		const bool opened = CascOpenStorage(path.c_str(), CASC_LOCALE_ALL, &handle);
+	void CASC::open(const fs::path& path, bool ptr) {
+		const bool opened = CascOpenStorage((path / (ptr ? ":w3t" : ":w3")).c_str(), CASC_LOCALE_ALL, &handle);
```

Comment:
Why not do this where CASC::open is called?

Reply:
This remained here from when I planned to use CascOpenStorageEx because I forgot path:productname is a valid way to supply product for multi-product CASC storages. Moved back to Hierarchy.cpp in next commit